src/test/java/unrc/dose/UserTest.java

```diff
@@ -92,4 +92,75 @@ public void userSet() {
 		assertEquals(name , pepe.get("username"));
 	}
 
-}
\ No newline at end of file
+	/**
+	 * delete an user
+	 * @result delete logically a user logged
+	 */
+	@Test
+	public void deleteUserSuccessfully(){
+		String username = "JohnConnor";
+		String password = "VeryHardPass";
+
+		assertEquals(true, User.deleteUser(username, password));
+	}
+
+	/**
+	 * delete Unsuccessfully user
+	 * @result user not found
+	 */
+	@Test
+	public void deleteUserUnsuccessfully(){
```

Comment:
Should say `Unsuccessful` because using `Unsuccessfully` gives the idea that the goal is to fail to login.